composablejs/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "composable",
```

Comment:
Is this available on npm repo?

Reply:
Yes it is, but I just saw that there's a `@` missing at the beginning.